Update README.md (#83)
CircleCI-Public/server-scripts: upgrade-postgres-to-14/README.md

# CircleCI Server: Postgres 12 → 14 upgrade script

> NOTE: This script is intended for server version 4.9 and after. We cannot guarantee support for earlier server versions.

Reviewer: ```suggestion
NOTE: This script is intended only for server version 4.9 and after. 
```
